scripts/serve-local.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+echo "starting..."
+
+set -o errexit
+
+TAG='dockless-api:local'
+echo "building docker image..."
+docker build --tag "$TAG" .
+echo "running docker image..."
+
+docker run \
+  --rm \
+  -it \
```

Comment:
I dropped the `-d` parameter for the `docker run` command because I prefer for the app to run in the foreground (not in detached mode) when I'm working in a dev environment. This is mainly so I can follow the app logs. Lmk if you have strong opinions on that.